include/quick-ftxui.hpp

```diff
@@ -128,7 +132,22 @@ struct node_printer : boost::static_visitor<> {
         : indent(indent), data(data_) {}
 
     void operator()(client::quick_ftxui_ast::expression const &expr) const {
-        ast_printer(data, indent + tabsize)(expr);
+        component_meta_data data_nest;
```

Comment:
Because screen is uninitialised it is a null pointer and this is a major issue.

Reply:
Ok, understood. I have made the changes here https://github.com/SAtacker/quick-ftxui/pull/11/commits/3e682ebaf06db1fe527688ffaf9af97dd197ada1